src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,35 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    font-size: 12px;
+    line-height: 14px;
+    font-weight: 500;
+    text-transform: uppercase;
+
+    margin-left: 20px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+
+    line-height: 60px;
+    color: $nav-link-color;
+    text-decoration: none;
+
+    &:hover {
+      transition: 0.3s;
```

Comment:
The transition property should be placed on the element itself, not on the hover state. This ensures that the transition effect occurs both when the mouse enters and when it leaves the element.